core/controllers/admin.py

```diff
@@ -656,6 +675,390 @@ def _generate_dummy_explorations(
         else:
             raise Exception('Cannot generate dummy explorations in production.')
 
+    def _create_dummy_interactions(self, state, dest):
```

Comment:
Should move this function to below where it's used.

Reply:
It's to maintain an organized class structure that a reader can expect. Generally, the more "public" and higher-order methods would be nearer the top since they may be relevant for files outside the current file and for other readers less familiar with that particular section of the codebase. If you're reading a code file top to bottom, then it's natural to abstract away the helper methods and place them lower so that the code gets more specific as you move down. Also, if you're navigating between files in a codebase and you're looking for functionality you can use, you don't have to dig so deep for the stuff you'd care about.